### ISSUE TITLE
Multi-figure grids

### ISSUE THREAD

**Issue by gabestein** [pubpub/pubpub#2084]
### What's your enhancement request?
Some way to create grids of multiple images/figures/etc., with a single caption, prepended by "Figures X-Y". Possibly with click-to-zoom-in on individual figures?

### What would this enhancement help you accomplish?
Right now folks use tables to do this (e.g. https://hdsr.mitpress.mit.edu/pub/oraonikr/release/1), which is less than ideal because it doesn't use the built-in caption font, has table outlines, and also isn't great on mobile. BUT, maybe we should use tables, and we should just fix them, especially on layout, and have the option to remove borders, and add captions?

---

### Any attachments or reference images? (Drag them here)

two-up:
<img width="718" alt="Screen Shot 2022-06-22 at 12 14 33" src="https://user-images.githubusercontent.com/639110/175081644-e6af0b52-8b5f-44ef-adfb-b607dee8418a.png">

four-up:
<img width="629" alt="Screen Shot 2022-06-22 at 12 16 00" src="https://user-images.githubusercontent.com/639110/175081933-ccdb863c-e162-4103-b61b-d0b067a188ae.png">

eight-up:
<img width="724" alt="Screen Shot 2022-06-22 at 12 15 10" src="https://user-images.githubusercontent.com/639110/175081758-f16e6d31-35c4-49b0-9466-95df0078e093.png">

**Comment by metasj** [comment]
Agreed that there's no good option that uses compact linespacing and the caption font (and centers the caption).  Perhaps a gallery function with a parameter for layout-templates (N-row, N-col, square?) that can have a table as one of the elements.
